src/ovp_pipeline/commands/_ui_renderers.py

```
@@ -3435,20 +3364,28 @@ def _cluster_href(*, limit_: int | None = None, show_all_: bool | None = None) -
 
     if show_all:
         limit_note = f" Showing all {total_count} clusters."
-    elif payload.get("is_limited"):
+    elif total_count > 0 and rendered_count > 0:
+        # Use 1-indexed range — "Showing 51–100 of 730" reads more
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)